spacy/cli/project/run.py

```diff
@@ -204,107 +256,3 @@ def validate_subcommand(
             ". ".join(help_msg),
             exits=1,
         )
-
```

Comment:
This code has been moved to `_util.py` as it is also called from  `parallel.py`.